### ISSUE TITLE
Migrate akka to pekko

### ISSUE THREAD

**Issue by laglangyue** [apache/incubator-pekko-platform-guide#55]
I found help wanted `https://pekko.apache.org/samples.html#help-wanted`, I am willing to do this.
the repo is just Document, I found that https://developer.lightbend.com/docs/akka-platform-guide It's already 404 and I don't know where this document will be displayed.

**Comment by pjfanning** [comment]
moved to https://developer.lightbend.com/docs/akka-guide/index.html

**Comment by laglangyue** [comment]
> moved to https://developer.lightbend.com/docs/akka-guide/index.html
Thanks,I will finish reading this document first and then take a look at this repo.
We haven't deployed this repo yet. Will we use GitHub Action to deploy it to `gitHub.io` or organize this document in other ways in the future

**Comment by pjfanning** [comment]
Publishing the site to the internet is a second task. The first task is to be able to change the docs and any code so that you can build it on your own computer and have it use Pekko references instead of Akka.

**Comment by laglangyue** [comment]
> Publishing the site to the internet is a second task. The first task is to be able to change the docs and any code so that you can build it on your own computer and have it use Pekko references instead of Akka.

Yes,  Should we migrate document tools to `Pekko SBT Paradox`?  Now this repo use `antora`, it seems to be different from other sample repos. Paradox can be easily built locally.

**Comment by pjfanning** [comment]
#24 is open - let's do that as a separate task later

Let's concentrate on change the code and doc content first

**Comment by laglangyue** [comment]
It seems that there hasn't been any progress for #24 yet. So we make modifications at the document and code levels firstly, and then don't have to wait for #24 ?

**Comment by pjfanning** [comment]
there is no reason to do #24 until we have this done

**Comment by laglangyue** [comment]
Atfer # 24, write markdown is better development experience, build locally and review the changes.
emm... get it,I will start this work later.
